app/viewmodels/DutyDefermentAccount.scala

```diff
@@ -16,14 +16,23 @@
 
 package viewmodels
 
-case class DutyDefermentAccount(accountNumber: String, statementsForAllEoris: Seq[DutyDefermentStatementsForEori],
+import java.time.LocalDate
+
+case class DutyDefermentAccount(accountNumber: String,
+                                statementsForAllEoris: Seq[DutyDefermentStatementsForEori],
                                 linkId: String, isNiAccount: Boolean) {
 
   val hasRequestedStatements: Boolean = statementsForAllEoris.exists(_.requestedStatements.nonEmpty)
   val hasCurrentStatements: Boolean = statementsForAllEoris.exists(_.currentStatements.nonEmpty)
 
   def firstPopulatedStatement: Option[DutyDefermentStatementsForEori] = statementsForAllEoris.find(_.groups.nonEmpty)
 
+  private val amtMonthsHistory: Int = 6
+  val monthsToDisplay: LocalDate = LocalDate.now().minusMonths(amtMonthsHistory)
+
+  def dropOldMonths(months: Seq[DutyDefermentStatementPeriodsByMonth]):
```

Comment:
Remove viewmodels from the return type and method could be written (method definition and return type can be on same line, method implementation on next line) like below 

  def dropOldMonths(months: Seq[DutyDefermentStatementPeriodsByMonth]): Seq[DutyDefermentStatementPeriodsByMonth] =
    months.dropRight(months.length - 5)